src/main.ts

```diff
@@ -3,23 +3,23 @@ import { getOctokit } from "@actions/github";
 import { yarnUpdateBase } from "./yarn-update-base";
 
 async function main(): Promise<void> {
-  const token = core.getInput("github-token", { required: true });
-  const context = core.getInput("context", { required: true });
+	const token = core.getInput("github-token", { required: true });
+	const context = core.getInput("context", { required: true });
 
-  const github = getOctokit(token, {});
+	const github = getOctokit(token, {});
 
-  console.log("Fabric Actions Script, context: " + context);
+	console.log("Fabric Actions Script, context: " + context);
 
-  switch (context) {
-    case "yarn-update-base":
-      yarnUpdateBase(
-        github.rest,
-        parseInt(core.getInput("issue-number", { required: true }))
-      );
-      break;
-    default:
-      throw new Error("Unknown context: " + context);
-  }
+	switch (context) {
+		case "yarn-update-base":
+			await yarnUpdateBase(
```

Comment:
ah, good spot, this await is needed 👍

Reply:
That was just the [`require-await`](https://eslint.org/docs/rules/require-await) rule, which wouldn't be caught by the recommended configuration.